docs/ocm-control-plane-walkthrough.md

```diff
@@ -0,0 +1,267 @@
+```
+
+session/window 2
+
+
+```
+kind export kubeconfig --name=mctc-workload-2 --kubeconfig=$(pwd)/local/kube/workload2.yaml && export KUBECONFIG=$(pwd)/local/kube/workload2.yaml
+```
+
+- register the spoke clusters
+
+In each window where you set the kubeconfig run the command you copied earlier. The name of the cluster is up to you. I used gateway-1 and gateway-2
+```
+example
+
+clusteradm join --hub-token the_token --hub-apiserver https://127.0.0.1:49710 --wait --cluster-name gateway-1 --force-internal-endpoint-lookup
+
+clusteradm join --hub-token the_token --hub-apiserver https://127.0.0.1:49710 --wait --cluster-name gateway-2 --force-internal-endpoint-lookup
+```
+Once the gateway clusters have registered you will need to accept them in the hub back in the terminal that has the hub cluster context set 
+
+```
+clusteradm accept --clusters gateway-1
+clusteradm accept --clusters gateway-2
+
+```
+
+### Setup a managedclusterset and bind it to the multi-cluster gateway controller namespace
+
+Now that we have registered the clusters and managed clusters, we can see them as resources in the hub
+
+```
+kubectl get managedclusters
+
+NAME        HUB ACCEPTED   MANAGED CLUSTER URLS                         JOINED   AVAILABLE   AGE
+gateway-1   true           https://mctc-workload-1-control-plane:6443   True     True        2m54s
+gateway-2   true           https://mctc-workload-2-control-plane:6443   True     True        2m37s
+
+```
+
+Let make these clusters usable by the gateway controller:
+
+In the hub cluster context execute the following commands:
+
+```
+k label managedcluster gateway-1 ingress-cluster=true
+k label managedcluster gateway-2 ingress-cluster=true
+
+```
+
+Next create the managed clusterset. This is a group of clusters that meet certain criterea. 
+
+```
+kubectl apply -f - <<EOF
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSet
+metadata:
+  name: gateway-clusters
+spec:
+  clusterSelector:
+    labelSelector: 
+      matchLabels:
+        ingress-cluster: "true"
+    selectorType: LabelSelector
+EOF
+```    
+
+Next bind this cluster set to our multi-cluster-traffic-controller-system namespace (setup with make local-setup)
+
+```
+kubectl apply -f - <<EOF
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSetBinding
+metadata:
+  name: gateway-clusters
+  namespace: multi-cluster-traffic-controller-system
+spec:
+  clusterSet: gateway-clusters
+EOF
+```
+
+
+### Create a placement for our gateways
+
+In order to place our gateways onto clusters, we need to setup a placement resource:
+
+```
+kubectl apply -f - <<EOF
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: http-gateway
+  namespace: multi-cluster-traffic-controller-system 
+spec:
+  numberOfClusters: 2
+  clusterSets:
+    - gateway-clusters
+EOF
+```    
+
```

Comment:
All of the above cmds are fine as a stopgap to get an env working with the code changes in this PR.
Let's move it all into the local-setup soon